### PR TITLE
Add minutes from two pa11y org meetings

### PR DESCRIPTION
**Pull request by andrewmee**
We could improve the formatting of these later - adding links, linking names, etc.

### CODE REVIEW

File: minutes/2016-05-06.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+## State of the projects:
+
+Agenda:
+- Command-Line
+- Dashboard/Webservice
+- Website
+- Discussing project status: as detailed
+- https://gist.github.com/rowanmanning/631161d2ea5d88a757afe1b0dbd21abc#file-status-md
+
+AM: Do we think we should be merging the dashboard and webservice?
```
Comment (hollsk) [inline minutes/2016-05-06.md:13]: you've lost your carriage returns on this discussion :(

File: minutes/2016-05-06.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+- Involving some designers?
+- Weekly/fortnightly/monthly meetings? (open to everyone)
+- Sponsorship by organisations (through dev time?)
+- How we work together (what with us all working all over the place now)
+
+RM is currently acting lead on everything, this could be split which would be more efficient and reduce RM’s workload.  Suggests nominating new leads for different parts of the project.
+
+Requires people who understand the project well.  Potentially if we want to have external people taking on these roles we might need to address ownership.
+
+AK: Where does it break down logically?  We need to decide that first.
```
Comment (hollsk) [inline minutes/2016-05-06.md:38]: Same goes for a lot of the discussions, actually, soz!

**Comment by hollsk** [comment]
Looks great, just needs some extra spaces at the end of lines for carriage returns^h^h^h^h^h^h^h^h^h^h^h^h that stuff you already did 👍

**Comment by andrewmee** [comment]
I've opted for new paragraphs for each conversation line at the moment.  Not sure if it might be more readable to use breaks instead...

**Pull request merged**